### PR TITLE
more fixes

### PR DESCRIPTION
**Pull request by kolharsam**
- [x] `schemaList` sql — fix, use `getRunSql`,
 - [x] take care of `additionalColumnsInfoQuery`
 - [x] handling fetching schema list should be done better — keep list of strings instead of objects, handle in all the places
 - [x] fixed minor db version issue on About Page

### CODE REVIEW

File: console/src/dataSources/services/mysql/sqlUtils.ts [hunk 1]
```diff
@@ -553,5 +554,14 @@ ORDER BY
...
 export const primaryKeysInfoSql = (options: { schemas: string[] }) =>
   getKeysSql('PRIMARY KEY', options);
+
 export const uniqueKeysSql = (options: { schemas: string[] }) =>
   getKeysSql('UNIQUE', options);
+
+const schemaListSQL = `SELECT schema_name FROM information_schema.schemata WHERE
+schema_name NOT IN (SELECT schema_name FROM information_schema.schemata
+WHERE schema_name = 'information_schema' OR schema_name = 'mysql' OR schema_name = 'sys'
+OR schema_name = 'performance_schema') ORDER BY schema_name ASC;`;
```
Comment (kolharsam) [inline console/src/dataSources/services/mysql/sqlUtils.ts]: The original rql query had the condition in order to place the nulls towards the end. Is that still necessary? @beerose

File: console/src/dataSources/services/mysql/sqlUtils.ts [hunk 2]
```diff
@@ -553,5 +554,14 @@ ORDER BY
...
+
+export const schemaList = (source: string) =>
+  getRunSqlQuery(schemaListSQL, source);
```
Comment (beerose) [inline console/src/dataSources/services/mysql/sqlUtils.ts]: Don't use it here. Export sql and use `getRunSqlQuery` in the place when it's needed. Same way as it's done for all other SQL utilities. we want SQL logic separated from API logic. Generating RQL queries (`getRunSqlQuery`) is an API related thing — thus it should not be here.

File: console/src/dataSources/services/mysql/sqlUtils.ts [hunk 3]
```diff
@@ -553,5 +554,14 @@ ORDER BY
 
 export const primaryKeysInfoSql = (options: { schemas: string[] }) =>
   getKeysSql('PRIMARY KEY', options);
+
 export const uniqueKeysSql = (options: { schemas: string[] }) =>
   getKeysSql('UNIQUE', options);
+
+const schemaListSQL = `SELECT schema_name FROM information_schema.schemata WHERE
```
Comment (beerose) [inline console/src/dataSources/services/mysql/sqlUtils.ts]: Why nested SQL? Those are some hard coded strings. Are you aware of the performance cost of this nested select? Please bring back the previous SQL.

File: console/src/dataSources/services/postgresql/index.tsx [hunk 4]
```diff
@@ -179,23 +180,13 @@ export const getGroupedTableComputedFields = (
...
-    `,
-  },
-};
+const schemaListSQL = `SELECT schema_name FROM information_schema.schemata WHERE
+schema_name NOT IN (SELECT schema_name FROM information_schema.schemata
+WHERE schema_name = 'information_schema' OR schema_name = 'pg_catalog' OR schema_name = 'hdb_catalog'
+OR schema_name = 'hdb_views' OR schema_name = 'pg_temp_1' OR schema_name = 'pg_toast_temp_1' OR
+schema_name = 'pg_toast') ORDER BY schema_name ASC;`;
+
+const schemaList = (source: string) => getRunSqlQuery(schemaListSQL, source);
```
Comment (beerose) [inline console/src/dataSources/services/postgresql/index.tsx]:
```suggestion
const schemaListSql = <SQL FROM ABOVE>`
```

**Review by beerose — CHANGES_REQUESTED** [review]

File: console/src/dataSources/services/postgresql/index.tsx [hunk 5]
```diff
@@ -179,23 +180,13 @@ export const getGroupedTableComputedFields = (
...
-    'hdb_catalog',
-    'hdb_views',
-    'pg_temp_1',
-    'pg_toast',
-    'pg_toast_temp_1'
-  ) order by schema_name asc;
-    `,
-  },
-};
+const schemaListSQL = `SELECT schema_name FROM information_schema.schemata WHERE
```
Comment (beerose) [inline console/src/dataSources/services/postgresql/index.tsx]: Same thing. Please rewrite this query.

**Review by beerose — APPROVED** [review]

**Pull request merged**